src/sass/components/_header.scss

```diff
@@ -224,3 +224,43 @@ header {
     }
   }
 }
+@include media-breakpoint-down(md) {
+  header {
+    .top-bar {
+      .text-left {
+        flex-grow: 2;
+
+        .header-first-li {
```

Comment:
advice for the future - always think if it's necessary to have such long selectors as `header .top-bar .text-left .header-first-li` and try to reduce the nesting by applying styles either directly to specific classes (eg. `.header-first-li { ...`) or with minimal nesting possible eg. `.top-bar .header-first-li { ...`

according to SCSS guidelines 3-4 levels of nesting should be maximum.